TODO.md

```diff
@@ -2,8 +2,6 @@
 
 ## Priority list
 
-- Testing
-  - Create a test user from a script that can be easily executed, configured and customised
 - Application
   - Protect proxy `/admin` endpoint (CSAPI-62)
```

Comment:
Can we remove this from the TODO list too?  (I forgot to do this after my PR)

Reply:
Thanks, Jon. That should be removed